app/views/registrations/_have_a_degree.html.erb

```diff
@@ -1,7 +1,8 @@
-<%= f.govuk_radio_buttons_fieldset(:degree, legend: { text: 'Do you have a degree?'} ) do %>
-  <%= f.govuk_radio_button :degree, 'yes', label: { text: 'Yes' } %>
-  <%= f.govuk_radio_button :degree, 'no', label: { text: 'No' } %>
-  <%= f.govuk_radio_button :degree, 'studying', label: { text: "I'm studying for a degree" } %>
-  <%= f.govuk_radio_button :degree, 'equivalent', label: { text: 'I have an equivalent qualification from another country' } %>
+<%= f.govuk_radio_buttons_fieldset(:degree, legend: { text: 'Do you have a degree?'},
+  hint_text: "You need to have a bachelor's degree, with honours." ) do %>
+  <%= f.govuk_radio_button :degree, '222750000', label: { text: 'Yes' } %>
+  <%= f.govuk_radio_button :degree, '222750004', label: { text: 'No' } %>
+  <%= f.govuk_radio_button :degree, '222750001', label: { text: "I'm studying for a degree" } %>
```

Comment:
These values don't look right - they should be mapping to degree type

Reply:
Have mapped to HaveADegree::OPTIONS[:yes]... etc

 the types are ApiClient::get_qualification_degree_status
 => [#<GetIntoTeachingApiClient::TypeEntity:0x00007f8f5a983d48 @id="222750000", @value="Graduate or postgraduate">, #<GetIntoTeachingApiClient::TypeEntity:0x00007f8f5a983730 @id="222750001", @value="Final year">, #<GetIntoTeachingApiClient::TypeEntity:0x00007f8f5a983348 @id="222750002", @value="Second year">, #<GetIntoTeachingApiClient::TypeEntity:0x00007f8f5a982e70 @id="222750003", @value="First year">, #<GetIntoTeachingApiClient::TypeEntity:0x00007f8f5a982a88 @id="222750005", @value="Other">, #<GetIntoTeachingApiClient::TypeEntity:0x00007f8f5a982498 @id="222750004", @value="I don't have a degree and am not studying for one">]